package.json

```diff
@@ -949,6 +949,7 @@
     "md5": "^2.1.0",
     "mdast-util-to-hast": "10.2.0",
     "memoize-one": "^6.0.0",
+    "mermaid": "^10.4.0",
```

Comment:
Is there a feature we require from v10? If we use `9.4.3` it avoids ESM and might help with package size.

Reply:
Thank you @Ikuni17, pushed here to see if there are any improvements if we use `9.4.3` https://github.com/elastic/kibana/pull/167873

Reply:
> Nice work Patryk. Couple questions. Is it possible to make a singular plugin to import into both cases and security solution?

Definitely, I think we could extract this plugin or even a whole editor to a separate package, but if you don't mind we could do it in the follow-up due to the Feature Freeze constraints

Reply:
@patrykkopycinski Thanks for trying. Maybe we can find a way to reduce size when extracting the plugin.